docs/root/install/building.rst

```diff
@@ -68,6 +68,9 @@ tests.
   organizations track and deploy master in production. We encourage you to do the same so that
   issues can be reported as early as possible in the development process.
 
+There are packaged Envoy pre-built binaries in rpm, deb, and Homebrew packages are available via
+`GetEnvoy.io <https://www.getenvoy.io/>`_.
+
 We will consider producing additional binary types depending on community interest in helping with
 CI, packaging, etc. Please open an `issue <https://github.com/envoyproxy/envoy/issues>`_ in GitHub
```

Comment:
Maybe add a GH link to getenvoy here and tell people to open issue against that repo for prebuilt binaries for different platforms? WDYT?

Reply:
SGTM, done